vars/cloudFoundryDeploy.groovy

```diff
@@ -145,7 +152,14 @@ def deployCfNative (config) {
             export HOME=${config.dockerWorkspace}
             cf login -u \"${username}\" -p '${password}' -a ${config.cloudFoundry.apiEndpoint} -o \"${config.cloudFoundry.org}\" -s \"${config.cloudFoundry.space}\"
             cf plugins
-            cf ${deployCommand} ${config.cloudFoundry.appName?:''} ${config.deployType == 'blue-green'?'--delete-old-apps':''} -f '${config.cloudFoundry.manifest}' ${config.smokeTest}"""
+            cf ${deployCommand} ${config.cloudFoundry.appName?:''} ${blueGreenDeployOptions} -f '${config.cloudFoundry.manifest}' ${config.smokeTest}"""
+        if (config.keepOldInstance) {
+            sh """#!/bin/bash
```

Comment:
It feels wrong to me to have those commands in another shell. The command could be added to the above script as needed (`if (config.keepOldInstance)`)?